File: go/complete-word-autokey-rofi/completeword/add.go
package completeword

import (
	"fmt"
	"os/user"
)

const addPath = "/.config/complete-word-autokey-rofi/words/add.txt"

func Add(selection string) {
	usr, _ := user.Current()
	home := usr.HomeDir
	words, err := wordsFromFile(home + addPath)
	if err != nil {
		fmt.Printf(err.Error())
	}
	word, err := ChooseWordUsingRofi(words, selection)
	if err != nil {
		fmt.Printf(err.Error())
	}
	writeWord(addPath, word)
	if err != nil {
		fmt.Printf(err.Error())
	}
}

File: autokey/add.py
import time

selection = ""
try:
    selection = clipboard.get_selection()
    time.sleep(0.1)
except:
    pass

word = selection
try:
    system.exec_command(
        "echo " + selection +
        " >> ~/.config/complete-word-autokey-rofi/words/added.txt")
    time.sleep(0.1)
except:
    pass

keyboard.send_keys("<right>")
